AmplifyPlugins/API/AWSAPICategoryPlugin/Operation/AWSHTTPURLResponse.swift

```diff
@@ -42,7 +42,9 @@ public class AWSHTTPURLResponse: HTTPURLResponse {
     }
 
     required init?(coder: NSCoder) {
-        fatalError("init(coder:) has not been implemented and is not expected to be called.")
+        self.body = nil
+        self.response = HTTPURLResponse()
```

Comment:
This should be decoding the incoming `response` on the coder, or else the decoded instance won't have any values from the decoded instance.

Reply:
Makes sense, I have to override the `encode(NSCoder)` to encode with keys that I expect to decode from the `init(NSCoder)` to decode correctly. updated code accordingly